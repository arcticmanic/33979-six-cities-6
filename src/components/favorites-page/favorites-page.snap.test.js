import React from 'react';
import {render} from '@testing-library/react';
import {Test} from '../../components/test/test-mock';
import FavoritesPage from './favorites-page';

test(`Favorite Page component render correctly`, () => {
  const {container} = render(
      <Test>
        <FavoritesPage />
      </Test>
  );
  expect(container).toMatchSnapshot();
});
