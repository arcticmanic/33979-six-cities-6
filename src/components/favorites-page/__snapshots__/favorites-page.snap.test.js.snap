// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Favorite Page component render correctly 1`] = `
<div>
  <div
    style="position: fixed; left: 50%; top: 50%; transform: translate(-50%, -50%);"
  >
    <div
      aria-busy="true"
      class=""
    >
      <svg
        aria-label="audio-loading"
        height="100"
        stroke="#4481c3"
        viewBox="0 0 38 38"
        width="100"
        xmlns="http://www.w3.org/2000/svg"
      >
        <g
          fill="none"
          fill-rule="evenodd"
        >
          <g
            stroke-width="2"
            transform="translate(1 1)"
          >
            <circle
              cx="18"
              cy="18"
              r="18"
              stroke-opacity=".5"
            />
            <path
              d="M36 18c0-9.94-8.06-18-18-18"
            >
              <animatetransform
                attributeName="transform"
                dur="1s"
                from="0 18 18"
                repeatCount="indefinite"
                to="360 18 18"
                type="rotate"
              />
            </path>
          </g>
        </g>
      </svg>
    </div>
  </div>
</div>
`;
